plyer/facades/sms.py

```diff
@@ -33,7 +33,7 @@ class Sms:
     Sms facade.
     '''
 
-    def send(self, recipient, message):
+    def send(self, recipient, message, mode='iMessage', **kwargs):
```

Comment:
`iMessage` is not something that could be supported on Android and iOS.
Better to keep `mode=None` here, and than set the default for macOS somewhere else.

Reply:
Ok, agreed. `kwargs.get(...)` inside `MacOSSMS` can pass the default value.

Reply:
Yep, but users will not notice the availability of the `mode` setting.

The suggestion is to keep `mode` set to a default value (but not `iMessage`), and to clarify where and how should be used on the docstring for the `send` method.

Users will refer to the following documentation when writing their apps🙂: 
https://plyer.readthedocs.io/en/latest/#plyer.facades.Sms

Reply:
@misl6 Done. Now `mode=None` is always passed into `MacOSSMS` but if its value remains `None` we will set it to `iMessage`.

Have also corrected docstring to reflect this change.